config/routes.rb

```diff
@@ -59,6 +59,8 @@
     get '/:platform/:name/contributors', to: 'projects#contributors', constraints: { :platform => PLATFORM_CONSTRAINT, :name => PROJECT_CONSTRAINT }
     get '/:platform/:name/:version/tree', to: 'tree#show', constraints: { :platform => /[\w\-]+/, :name => PROJECT_CONSTRAINT, :version => VERSION_CONSTRAINT }, as: :version_tree
     get '/:platform/:name/:version/dependencies', to: 'projects#dependencies', constraints: { :platform => PLATFORM_CONSTRAINT, :name => PROJECT_CONSTRAINT, :version => VERSION_CONSTRAINT }
+    get '/:platform/:name/:version/dependencies_only', to: 'projects#dependencies_only', constraints: { :platform => PLATFORM_CONSTRAINT, :name => PROJECT_CONSTRAINT, :version => VERSION_CONSTRAINT }
+    post '/projects/dependencies_only', to: 'projects#dependencies_only_bulk'
```

Comment:
If we make the single package version a query string, this then becomes `/projects/dependencies` which feels nicer too